libevm/VMOpt.cpp

```diff
@@ -119,9 +126,10 @@ void VM::optimize()
 		}
 #endif
 	}
-	
-	copyCode(pc - nBytes + 33);
 
+	if (pc > nBytes)
+		resizeCodeSpace(m_codeSpace.size() + pc - nBytes);
+	
```

Comment:
I think this is not needed at all, as noted above.  Was the reason for creating resizeCodeSpace() so we could reuse it here?